stable_baselines/common/evaluation.py

```diff
@@ -0,0 +1,39 @@
+import numpy as np
+
+
+def evaluate_policy(model, env, n_eval_episodes=10, deterministic=True,
```

Comment:
Handy function, but could it also return a list/ndarray of per-episode rewards in case user wants to calculate more statistics (max, median, min, quantiles or even full distribution), perhaps a flag to switch the behavior?